packages/grafana-schema/src/index.ts

```diff
@@ -5,3 +5,9 @@
  */
 export * from './veneer/common.types';
 export * from './index.gen';
+
+// TODO: The following imports should be included in ./index.gen
```

Comment:
This lets our current system work as expected.  Given that we are looking to export this into a more more nested structure with `Resource<Metadata,Spec,Status>` wrappers anyway, I think this is good enough for this PR.